db/seeds.rb

```diff
@@ -1,7 +1,37 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
 # Examples:
-#
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+User.create([
```

Comment:
insert_all is probably faster

Reply:
You're rigth! I did some test with benchmark

![image](https://user-images.githubusercontent.com/15331175/150627077-de368a84-5c50-4959-8b9e-4cdde44b27b9.png)